src/stellar/stellarEngine.js

```diff
@@ -499,13 +504,13 @@ export class StellarEngine extends CurrencyEngine {
       if (!transaction) {
         throw new Error('ErrorInvalidTransaction')
       }
-      this.log('Signing...')
+      this.log.warn('Signing...')
       const keypair = this.stellarApi.Keypair.fromSecret(
         this.walletInfo.keys.stellarKey
       )
       await transaction.sign(keypair)
     } catch (e) {
-      this.log(e)
+      this.log.error(e)
```

Comment:
Add a line that logs the fact that you are in signTx. ie

      this.log.error(`ERROR signTx ${JSON.stringify(e)}`)